python/pyspark/worker.py

```diff
@@ -160,6 +166,24 @@ def read_udfs(pickleSer, infile, eval_type):
 
 
 def main(infile, outfile):
+    if raven:
```

Comment:
This adds overhead to all instances but we only seem to use this if an exception is thrown (or does the raven client record other information if its set up?)

Reply:
This adds tiny overhead for worker startup, but not such to care about. The main overhead comes on exception catch and send it to Sentry (HTTP request and traceback formation and etc.), but at that moment you don't actually care about speed since code on worker is already broken and won't be executed anymore.

Reply:
I mean does this overhead give us anything? Like do we need raven set up before the exception occurs? For something like streaming folks have done a bunch of work to get the worker startup time down and I'd rather not add things in the hot path if we don't need to (even if they are small). Of course if its needs to be there that's a different discussion.

Reply:
Ah, I get your point. Well, indeed, it's possible to move that branch into the `except Exception` branch. In the end, if exception happens, pyspark worker will get terminated, so raven client is for one time usage and no need to keep it around all the time.

Reply:
> I'd rather not add things in the hot path if we don't need to

+1